cli/demo_teeracle_whitelist.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Demo to show that an enclave can update the exchange rate only when
+#   1. it is a registered enclave
+#   2. and that the code used is reliable -> the enclave is in the teeracle whitelist.
```

Comment:
Could you add here that the sudo account needs to add the enclave into the whitelist?